.travis.yml

```diff
@@ -41,7 +43,7 @@ install:
       conda install -c conda-forge gxx_linux-64;
     fi
   - conda install --yes --file=python/requirements/conda-minimal.txt
-  - conda install -c bioconda --yes pysam
+  #- conda install -c bioconda --yes pysam
```

Comment:
How are we able to do without pysam? Ah, I see, we're probably getting it from pip. Well, that'll be brittle I'm guessing, and it's a heck of a big compile if they don't have binary wheels built.

Might be easier to just bump to 3.8 on Appveyor and leave the travis config alone? We're already excluding a bunch of awkward stuff on Windows anyway.

Reply:
This line is commented out (was planning to remove but wanted the discussion) because pysam via conda doesn't support 3.8. You're right that it is coming in from pip later on.
Build log on linux:
`Using cached pysam-0.15.4-cp38-cp38-manylinux2010_x86_64.whl (12.0 MB)`
And OSX:
`Downloading pysam-0.15.4-cp36-cp36m-macosx_10_6_intel.whl`

So I think this is OK.